io.itch.Itch.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
the upstream binary doesn't come with a .destkop? The launcher doesn't have a launcher?

Reply:
They dynamically generate it [here](https://github.com/itchio/itch-setup/blob/a14538db96c1f5c241c85beaa5c926e53b9626f0/native/native_linux.go#L472)

Let me try and get this to match a bit better